docfx_yaml/extension.py

```diff
@@ -1384,12 +1392,51 @@ def _find_and_add_summary_details(
     if summary_type in [CLASS]:
         name_to_use = f"[{uid}]({cgc_url}{uid})"
 
-    _ENTRY_NAME_AND_ENTRY_CONTENT_BY_SUMMARY_TYPE[summary_type][1].append({
-        "uid": uid,
+        _ENTRY_NAME_AND_ENTRY_CONTENT_BY_SUMMARY_TYPE[summary_type][1].append({
+            "uid": uid,
+            "name": name_to_use,
+            "fullName": uid,
+            "isExternal": False,
+        })
+        return
+
+    # if summary_type in [METHOD, PROPERTY]:
```

Comment:
You can make this explicit with an `elif` statement, also you wouldn't need to return early above.

Reply:
Chatted offline, keeping as-is.